content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -455,9 +455,7 @@ $ kubectl exec deploy/sleep -- sh -c "for i in \$(seq 1 100); do curl -s http://
 To remove the `productpage-viewer` authorization policy, waypoint proxies and uninstall Istio:
 
 {{< text bash >}}
-$ kubectl delete authorizationpolicy productpage-viewer
```

Comment:
Shouldn't we keep this, especially given the sentence immediately before this mention's removing the authorization policy?

Reply:
`istioctl uninstall -y --purge` does the same thing by removing the authz policy CRD. Mentioning it as a separate step seems redundant to me. Maybe I can revise the words as well.

Reply:
Whops I missed the uninstall command below the waypoint delete.